src/main/java/frc/robot/Robot.java

```diff
@@ -18,6 +18,7 @@
  * project.
  */
 public class Robot extends TimedRobot {
+  //TODO: Figure out how to execute commands
```

Comment:
Sorry, I'm not sure what this means?

Reply:
Hi Anthony, I was just trying a todo for me to figure out how to run commands when the operator presses buttons.

Reply:
Then please remove the comment, it's in the wrong class first off and you made the button presses, it's not to do if it's been done. It would be confusing to have random TODOs that have already been done.

Reply:
Change added